front-end/peer-prep/src/util/util.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
Is react import required here? Not very familiar with react, can @Acerizm verify?

Reply:
Based on this file don't need